src/v/config/configuration.h

```diff
@@ -55,6 +55,11 @@ struct configuration final : public config_store {
     property<std::size_t> coproc_max_batch_size;
     property<std::chrono::milliseconds> coproc_offset_flush_interval_ms;
 
+    // V8 engine
+    property<bool> enable_v8;
```

Comment:
Is the `enable_coproc` flag not good enough? Because you'll also need it set to `true` to obtain new coprocessors from `rpk wasm deploy`. Might as well all operate under the same feature flag

Reply:
I create event_handler for data_policy, and do not need coproc